charts/opentelemetry-demo/values.yaml

```diff
@@ -4,6 +4,24 @@ observability:
   jaeger:
     enabled: true
 
+default:
```

Comment:
I think we should supply the option to allow downard api to be used in env vars and also a way to set more values in `OTEL_RESOURCE_ATTRIBUTES`, but unless the demo was changed to require these I don't think they should be default.

Reply:
got it.